test/instrumentation/modules/cassandra-driver/cassandra-driver.test.js

```diff
@@ -0,0 +1,441 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict';
+
+if (process.env.GITHUB_ACTIONS === 'true' && process.platform === 'win32') {
+  console.log('# SKIP: GH Actions do not support docker services on Windows');
+  process.exit(0);
+}
+
+const test = require('tape');
+const semver = require('semver');
+const { validateSpan } = require('../../../_validate_schema');
+const {
+  runTestFixtures,
+  safeGetPackageVersion,
+  sortApmEvents,
+} = require('../../../_utils');
+
+const CASSANDRA_VERSION = safeGetPackageVersion('cassandra-driver');
+const TEST_KEYSPACE = 'mykeyspace';
+const TEST_TABLE = 'myTable';
+const TEST_DATACENTER = 'datacenter1';
+const TEST_USE_PROMISES = String(semver.satisfies(CASSANDRA_VERSION, '>=3.2'));
+
+const testFixtures = [
+  {
+    name: 'cassandra-driver simple usage for versions <4.7.0',
+    script: 'fixtures/use-cassandra-driver.js',
+    cwd: __dirname,
+    timeout: 20000, // sanity guard on the test hanging
+    maxBuffer: 10 * 1024 * 1024, // This is big, but I don't ever want this to be a failure reason.
+    env: {
+      TEST_KEYSPACE,
+      TEST_TABLE,
+      TEST_DATACENTER,
+      TEST_USE_PROMISES,
+    },
+    versionRanges: {
+      // v4.7.0 is a bad release for node versions <16.9
+      // and we want to test form 14.7 and above
+      node: '<16.9',
```

Comment:
Oh, actually, this `node: ...` condition could be dropped here, right? We can test *old* cassandra-driver releases with newer node versions.

Reply:
Thats correct. I'll drop it :)